Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -1155,7 +1157,7 @@ WarpX::MacroscopicEvolveE (int lev, PatchType patch_type, amrex::Real a_dt, amre
         m_fields.get_alldirs(FieldType::Efield_fp, lev),
         m_fields.get_alldirs(FieldType::Bfield_fp, lev),
         m_fields.get_alldirs(FieldType::current_fp, lev),
-        m_fields.get_alldirs(FieldType::edge_lengths, lev),
+        m_eb_update_E[lev],
```

Comment:
I'm assuming the plan is to replace these arguments once `iMultiFabRegister` is introduced?

Reply:
Yes, I agree.